sawtooth_sdk/messaging/exceptions.py

```diff
@@ -24,3 +24,11 @@ def __init__(self):
 class WorkloadConfigurationError(Exception):
     def __init__(self):
         super().__init__("A workload object is not set.")
+
+
+# ValidatorVersionError is used internal to the sdk, and
+# any other use can cause undesirable or unexpected behavior.
+class ValidatorVersionError(Exception):
+    def __init__(self):
+        super().__init__("the SDK version doesn't match with that of "
+                "validator")
```

Comment:
This line should have 4 more spaces. Originally this commit was part of sawtooth-core, but the python SDK is moved out now. This lint error came in when copy pasting code from earlier commit to new repository.